src/net.h

```diff
@@ -376,7 +380,8 @@ class CNode
     bool fOneShot;
     bool fClient;
     bool fInbound;
-    bool fNetworkNode;
+    bool fOutbound;    // any outbound node not connected with -addnode, connect-thinblock or -connect
```

Comment:
I strongly suggest not using a known and and well defined term 'outbound' to mean something different here.

Maybe rename this variable to something like `fAutoInitiated`.